RATapi/examples/non_polarised/DSPC_standard_layers.ipynb

```diff
@@ -0,0 +1,536 @@
+      "|   1   | SMW  | dspc_bil_smw | SMW Background |        add        | Silicon |   SMW    | Scalefactor 2 | Resolution 1 |  False   |     Oxide     |\n",
+      "|       |      |              |                |                   |         |          |               |              |          |   SAM Tails   |\n",
+      "|       |      |              |                |                   |         |          |               |              |          |   SAM Heads   |\n",
+      "|       |      |              |                |                   |         |          |               |              |          | Central Water |\n",
+      "|       |      |              |                |                   |         |          |               |              |          | Bilayer Heads |\n",
+      "|       |      |              |                |                   |         |          |               |              |          | Bilayer Tails |\n",
+      "|       |      |              |                |                   |         |          |               |              |          | Bilayer Tails |\n",
+      "|       |      |              |                |                   |         |          |               |              |          | Bilayer Heads |\n",
+      "+-------+------+--------------+----------------+-------------------+---------+----------+---------------+--------------+----------+---------------+\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(problem)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "136e2c63-f439-4c2d-bb10-453950bbf41b",
+   "metadata": {},
+   "source": [
+    "## Running the Project\n",
+    "\n",
+    "To run a project in RAT, we first need to define a controls block:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "c9ec7e39-48a8-4651-b74c-19d5800c67d7",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "+------------------+-----------+\n",
+      "|     Property     |   Value   |\n",
+      "+------------------+-----------+\n",
+      "|    procedure     | calculate |\n",
+      "|     parallel     |   single  |\n",
+      "| calcSldDuringFit |   False   |\n",
+      "| resampleMinAngle |    0.9    |\n",
+      "| resampleNPoints  |     50    |\n",
+      "|     display      |    iter   |\n",
+      "+------------------+-----------+\n"
+     ]
+    }
+   ],
+   "source": [
+    "controls = RAT.Controls(display='iter')\n",
+    "print(controls)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05f44162-c5b2-46e4-9233-b63e81089fd8",
+   "metadata": {},
+   "source": [
+    "The default action (\"procedure\") is just a single calculation. So call RAT.run() with this, and then plot our our initial starting position:"
```

Comment:
our starting position for what? we don't do anything with this after this cell? 

(also is 'initial starting position' a redundant phrase?)

Reply:
The matlab version goes on to do Bayes stuff, but we're leaving it here for now.